aspnetcore/blazor/forms-and-input-components.md

```diff
@@ -3787,6 +3857,27 @@ Create a form using the normal HTML `<form>` tag and specify an `@onsubmit` hand
 
 To submit a form based on another element's DOM events, for example `oninput` or `onblur`, use JavaScript to submit the form ([`submit` (MDN documentation)](https://developer.mozilla.org/docs/Web/API/HTMLFormElement/submit)).
 
+Enhanced navigation is supported with the *explicit addition* of the `data-enhance` HTML attribute.
+
+<span aria-hidden="true">❌</span><span class="visually-hidden">Unsupported:</span> You can't set enhanced navigation on a form's ancestor element to enable enhanced navigation for the form.
+
+```html
+<div data-enhance>
+    <form ...>
+        <!-- NOT enhanced -->
+    </form>
+</div>
+```
+
+> [!WARNING]
+> ***Don't try to enhance form navigation for forms that POST to non-Blazor endpoints.***
+>
+> This is the main reason why enabling form POSTs with enhanced navigation requires an explicit gesture on each `<form>` element. It isn't safe to assume that all forms in general should use enhanced posts because any given form might not target a Blazor endpoint.
```

Comment:
Same comment about this warning as the one above. I want to avoid making it sound like enhanced form handling is somehow "unsafe".